src/internal/components/selectable-item/styles.scss

```diff
@@ -34,7 +34,8 @@
     border-block-end-color: transparent;
   }
 
-  &:not(:first-child) {
+  &:not(:first-child),
+  &.virtual {
```

Comment:
Since we moved the "strut" in the virtual list implementation below the options (see changes in `select/parts/virtual-list.tsx`, this selector was not matching the first option anymore, therefore added `&.virtual` for the virtual list case.